docs/sdk-docs/installing-dapp/README.md

```diff
@@ -0,0 +1,40 @@
+# Install and Run the dApp
+
+Clone the Raiden Light Client repository.
+
+```bash
+git clone --recurse-submodules https://github.com/raiden-network/light-client.git
+```
+
+## Build the SDK
+
+Change to the `raiden-ts` directory, install the dependencies and build the SDK.
+
+```bash
+cd ../raiden-ts
```

Comment:
Not `cd light-client/raiden-ts`?

Reply:
This assumes that you are already in the directory where you cloned the project. Maybe add the information about moving to the directory where the project was cloned after the `git clone` command

Reply:
I would make it 5yo-proof and write down the exact steps :)

Besides that even if you are already in the cloned project, `cd ../raiden-ts` would not work